src/carousel.scss

```diff
@@ -0,0 +1,155 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-carousel;
+$button: #{$fd-namespace}-button;
+
+.#{$block} {
+  $fd-carousel-page-indicator-border: 0.0625rem solid var(--sapPageFooter_BorderColor) !default;
+  $fd-carousel-button-size: 2.125rem !default;
+  $fd-carousel-button-content-offset: 0.5rem !default;
+  $fd-carousel-dot-size: 0.25rem !default;
+  $fd-carousel-dot-active-size: 0.5rem !default;
+
+  @include fd-reset();
+  @include fd-flex(column);
+
+  width: 100%;
+  min-width: 15.5rem;
+  max-width: 100%;
+  height: 100%;
+
+  &__content {
+    @include fd-reset();
+
+    flex: 1 1 auto;
+    width: 100%;
+    background: var(--sapBackgroundColor);
+    position: relative;
+
+    .#{$block}__button.#{$button} {
+      box-shadow: var(--sapContent_Shadow1);
+      position: absolute;
+      top: calc(50% - 1.0625rem);
+      margin: 0;
+
+      &.sap-icon--slim-arrow-left {
+        left: $fd-carousel-button-content-offset;
+
+        @include fd-rtl() {
+          left: initial;
+          right: $fd-carousel-button-content-offset;
+        }
+      }
+
+      &.sap-icon--slim-arrow-right {
+        right: $fd-carousel-button-content-offset;
+
+        @include fd-rtl() {
+          right: initial;
+          left: $fd-carousel-button-content-offset;
+        }
+      }
+    }
+  }
+
+  &__item {
+    @include fd-reset();
+    @include fd-flex-center();
+
+    width: 100%;
+    height: 100%;
+    display: none;
+
+    &--active {
+      display: block;
+    }
+  }
+
+  &__page-indicator-container {
+    @include fd-reset();
+    @include fd-flex-center();
+
+    width: 100%;
+    height: 100%;
+    min-height: 2.75rem;
+    max-height: 3.5rem;
+    background: var(--sapPageFooter_Background);
+    padding: 0.375rem 0.5rem;
+    border-top: $fd-carousel-page-indicator-border;
+
+    &:first-child {
+      border-top: none;
+      border-bottom: $fd-carousel-page-indicator-border;
+    }
+  }
+
+  &__page-indicators {
+    @include fd-reset();
+
+    @include fd-flex-center() {
+      flex-wrap: wrap;
+    }
+
+    width: 9rem;
+    padding: 0.5rem;
+  }
+
+  &__page-indicator {
+    @include fd-reset();
+
+    list-style-type: none;
```

Comment:
Don't we have some mixin for reseting list styles?

Reply:
I can't find such mixin.